src/__tests__/store/reducers/apiList.reducer.test.js

```diff
@@ -5,62 +5,143 @@ import {
+        const result = apiList({}, { type: FETCH_ENDPOINTS_SUCCESS, payload: randomString });
 
-        expect(result.apiList).toEqual(randomString);
-        expect(result.isFetching).toEqual(false);
+        it('returns fetched endpoint success state', () => {
+            expect(result.apiList).toEqual(randomString);
+            expect(result.isFetching).toEqual(false);
+        });
+
+        it('should handle only change exact amount of reducer properties', () => {
+            expect(touchedReducerProps(result)).toBe(2);
+        });
     });
 
-    it('returns discarded pagination state', () => {
-        const result = apiList(initialState, { type: DISCARD_PAGINATION });
+    describe('DISCARD_PAGINATION', () => {
+        const result = apiList({}, { type: DISCARD_PAGINATION });
+
+        it('returns discarded pagination state', () => {
+            expect(result.currentPageIndex).toEqual(0);
+        });
 
-        expect(result.currentPageIndex).toEqual(0);
+        it('should handle only change exact amount of reducer properties', () => {
+            expect(touchedReducerProps(result)).toBe(1);
+        });
     });
 
-    it('returns set pagination state', () => {
+    describe('SET_PAGINATION_PAGE', () => {
         const pageIndex = 1;
-        const result = apiList(initialState, { type: SET_PAGINATION_PAGE, payload: pageIndex });
+        const result = apiList({}, { type: SET_PAGINATION_PAGE, payload: pageIndex });
 
-        expect(result.currentPageIndex).toEqual(pageIndex);
+        it('returns set pagination state', () => {
+            expect(result.currentPageIndex).toEqual(pageIndex);
+        });
+
+        it('should handle only change exact amount of reducer properties', () => {
+            expect(touchedReducerProps(result)).toBe(1);
+        });
     });
 
-    it('returns resfresh endpoints state', () => {
+    // TODO: investigate for more better solution
+    describe('REFRESH_ENDPOINTS', () => {
         const result = apiList(initialState, { type: REFRESH_ENDPOINTS });
 
-        expect(result.apiList).toEqual([]);
+        it('returns resfresh endpoints state', () => {
+            expect(result.apiList).toEqual([]);
+        });
+
+        it('returns resfresh endpoints state when having already elements on apiList', () => {
+            const newInitialState = { ...initialState, apiList: [
+                {name: 1},
+                {name: 2},
+                {name: 3},
+            ]};
+            const payload = 3;
+            const result = apiList(
+                newInitialState,
+                {
+                    type: REFRESH_ENDPOINTS,
+                    payload,
+                },
+            );
+
+            expect(result.apiList).toEqual([
+                {name: 1},
+                {name: 2},
+            ]);
+        });
+
+        it('should handle only change exact amount of reducer properties', () => {
```

Comment:
You can omit the "should" word in the descriptions. For instance, this one can be written as:
`handles only change the exact amount of reducer properties`. That way you are not repeating yourself writing "should" everywhere.

Reply:
gracias Señor ! 👍